tests/Integrations/Lumen/V5_2/TraceSearchConfigTest.php

```diff
@@ -30,14 +32,14 @@ public function testScenario()
             $this->call(GetSpec::create('Testing trace analytics config metric', '/simple'));
         });
 
-        $this->assertExpectedSpans(
+        $this->assertFlameGraph(
             $traces,
             [
                 SpanAssertion::build(
                     'lumen.request',
                     'lumen',
                     'web',
-                    'App\Http\Controllers\ExampleController@simple simple_route'
+                    'GET simple_route'
```

Comment:
A note here. `TraceSearchConfigTest.php` were not run in composer.json as part of the `composer lumen-*-test` suites. We had moved from `Controller@action route` to `GET route` + `tag: Controller@action` from a long time but we did not realized that tests were outdated